pkg/config/authorization.go

```diff
@@ -48,3 +76,9 @@ func (config *AuthorizationConfig) GetPriority() int {
 func (config *AuthorizationConfig) GetConditions() []common.JSONPatternMatchingRule {
 	return config.Conditions
 }
+
+// impl:metrics.Object
+
+func (config *AuthorizationConfig) Measured() bool {
```

Comment:
Just an idea, measured to me sounds more like something related to size or distance, maybe `exportMetrics` would be easier to read?

Reply:
Changed to `MetricsEnabled()`.